text/chapter3.md

```diff
@@ -561,7 +561,7 @@ This type signature says that `findEntry` takes two strings, the first and last
 And here is the definition of `findEntry`:
 
 ```haskell
-findEntry firstName lastName book = head $ filter filterEntry book
+findEntry firstName lastName book = head (filter filterEntry book)
```

Comment:
Since there's a bunch of content to get through before the explanation of `$`, I figured it would be better to reveal the alternative version later on.